index.d.ts

```diff
@@ -8,4 +8,5 @@ declare module 'react-native-dotenv' {
   export const IOS_GOOGLE_CLIENT_ID: string
   export const CLIENT_ID: string
   export const APP_ENV: string
+  export const SERVER_URL: string
```

Comment:
We already have `API_BASE_URL`. What's the purpose of this?

Reply:
forgot to remove this after i realised we had `API_BASE_URL`, removed